usagov/vote-gov-drupal
# VOTE-1987: Add custom module to block site_builders on production.

## Jira ticket

[VOTE-1987](https://cm-jira.usa.gov/browse/VOTE-1987)

## Description

Add custom module to block users who have the site_builder role on production.

## Deployment and testing

### Post-deploy steps

n/a

### QA/Testing instructions

1. Locally enable the Vote Users module.
2. Login as an admin and go to the people admin page.
3. Confirm that users who have the Site Builder role all have a Blocked status.
4. To test a login attempt you can look up the uid of a site builder user and run 'lando drush uli uid={uid}'.
5. Confirm that the response is " Account user.name is blocked and thus cannot login. The user:unblock command may be helpful. "
6. Disable the Vote Users module.
7. Confirm that users who have the Site Builder role all have a Active status.

## Checklist for the Developer




## Checklist for the Peer Reviewers
